.drone.yml

```diff
@@ -2376,7 +2376,7 @@ steps:
   - package
 
 - name: end-to-end-tests
-  image: grafana/ci-e2e:12.19.0-1
+  image: grafana/ci-e2e:12.18-1
```

Comment:
Should this be downgraded?

Reply:
👀 That's weird

Reply:
@sakjur Does it also happen if you re-generate .drone.yaml?

Reply:
Yes, I even deleted `.drone.yml` before generating it to make sure it didn't have any artifacts.

And it makes sense, because `lib.star` points to 12.18, so seems like it could be something like an accidental backport 🤔

https://github.com/grafana/grafana/blob/v7.3.x/scripts/lib.star#L608

We can go up or stay down, but I'm not sure which one we should go for.

Reply:
I'll check.

Reply:
@sakjur I think we can safely upgrade. The difference is just a bump of Node minor version and installing of Git into the image.